docs-site/content/0.25.0/api/vector-search.md

```diff
@@ -310,216 +342,1504 @@ curl "http://localhost:8108/collections/docs/documents" -X POST \
+        }
+      }
+    }'
+
+```
+   </template>
+
+</Tabs>
+
+### Remote Embedding API parameters
+
+You can use any of the following parameters to fine-tune how API calls are made to remote embedding services:
+
+#### During Search
+
+| Parameter                     | Description                                                                                               |
+|-------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `remote_embedding_timeout_ms` | How long to wait until an API call to a remote embedding service is considered a timeout, during a search |
+| `remote_embedding_num_tries`  | The number of times to retry an API call to a remote embedding service on failure, during a search        |
+
+<Tabs :tabs="['JavaScript','PHP','Python','Shell']">
+
+  <template v-slot:JavaScript>
+
+```js
+let search_parameters = {
+  'q'                          : 'chair',
+  'query_by'                   : 'embedding',
+  'prefix'                     : false,
+  'remote_embedding_timeout_ms': 5000,
+  'remote_embedding_num_tries'   : 3
+}
+
+client.collections('products').documents().search(search_parameters)
+```
+
+  </template>
+
+<template v-slot:PHP>
+  
+```php
+$search_parameters = [
+  'q'                          => 'chair',
+  'query_by'                   => 'embedding',
+  'prefix'                     => false,
+  'remote_embedding_timeout_ms'=> 5000,
+  'remote_embedding_num_try'   => 3
+];
+
+$client->collections['products']->documents->search($search_parameters);
+```
+
+  </template>
+
+<template v-slot:Python>
+  
+  ```py
+search_parameters = {
+  'q'                          : 'chair',
+  'query_by'                   : 'embedding',
+  'prefix'                     : false,
+  'remote_embedding_timeout_ms': 5000,
+  'remote_embedding_num_try'   : 3
+}
+
+client.collections['products'].documents.search(search_parameters)
+```
+
+  </template>
+
+<template v-slot:Shell>
+    
+  ```bash
+curl --location 'http://localhost:8108/collections/products/documents/search?q=chair&query_by=embedding&prefix=false&remote_embedding_timeout_ms=5000&remote_embedding_num_try=3' \
+--header 'X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}'
+```
+
+  </template>
+
+</Tabs>
+
+
+#### During Indexing
+
+| Parameter                     | Description                                                                                                                                                                           |
+|-------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `remote_embedding_batch_size` | Max size of each batch that will be sent to remote APIs while importing multiple documents at once. Using lower amount will lower timeout risk, but increase number of requests made. |
```

Comment:
Could you add a new column here called `Default` and add the default value for each parameter?